src/components/header.tsx

```diff
@@ -154,7 +155,7 @@ function Header(props: HTMLChakraProps<'header'>) {
       width='full'
       {...props}
     >
-      <chakra.div height='4.5rem' mx='auto' maxW='8xl'>
+      <chakra.div height='4.5rem' mx='auto' maxW={maxW} maxWidth={maxWidth}>
```

Comment:
Double max width props?

Reply:
As I needed to pass a maxWidth to this for the tutorial Layout, I've built it like this. People can then use either the short `maxW` or `maxWidth` to achieve this.